docs/blog/_posts/2017-05-31-first-dotty-milestone-release.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.1.2-RC1, a major step towards Scala 3
+author: Dmytro Petrashko
+authorImg: /images/petrashko.jpg
+---
+
+Today, we excited to release Dotty version 0.1.2-RC1.  This release
+serves as a technology preview that demonstrates new language features
+and the compiler supporting them.
+
+
+<!--more-->
+
+## Why is this important?
+
+_Dotty_ is the project name for a language and compiler that
+is slated to become Scala 3.0. This is an ongoing development,
+developed transparently as open source software. The Dotty project
```

Comment:
transparently developed (also avoids the immediate repetition of "development" followed by "developed")